.git-blame-ignore-revs

```diff
@@ -153,3 +153,6 @@ bdfde18037f8d9f9b641a4016c8ada4dc4cbf856
 
 # nixos/ollama: format with nixfmt-rfc-style (#329561)
 246d1ee533810ac1946d863bbd9de9b525818d56
+
+# steam: cleanup (#216972)
+ad815aebfbfe1415ff6436521d545029c803c3fb
```

Comment:
Isn't necessary anymore, is it?

Reply:
This is an old commit that annoyed me while debugging.

Reply:
Oh, OK.